h/static/scripts/util/search-text-parser.js

```diff
@@ -122,7 +116,84 @@ function getLozengeValues(queryString) {
   };
 }
 
+/**
+ * Return true if the input query term string has a known named
+ * query term.
+ *
+ * @param {String} queryTerm The query term string.
```

Comment:
Nit: For consistency with the rest of our documentation, primitive types should be written in lowercase: `string`, `boolean`, `number`. In other words, what `typeof value` returns for primitive types.